### PR TITLE
feat(scheduler): expose exports file

### PR DESCRIPTION
**Pull request by justinorringer**
### Description

[RHCLOUD-XXXXX](https://issues.redhat.com/browse/RHCLOUD-XXXXX)
Exposes the export options through chrome-service. Follows pattern from other files.

---

### How to test locally

---

### Anything reviewers should know?

---

### Checklist
- [ ] Tests: new/updated tests cover the change
- [ ] API: spec updated if endpoints changed (or N/A)
- [ ] Migrations: backwards compatible if schema changed (or N/A)
- [ ] Dependencies: no known impact to dependent services
- [ ] Security: reviewed against [secure coding checklist](https://github.com/RedHatInsights/secure-coding-checklist) (or N/A)

## Summary by Sourcery

Expose export configuration from environment through chrome configuration generation and persist it as a static exports JSON file.

New Features:
- Generate an exports configuration file from the FEO_EXPORTS environment variable for use by the chrome service.

Deployment:
- Wire the FEO_EXPORTS environment variable from the frontend context ConfigMap in the ClowdApp deployment configuration.

Tests:
- Add unit tests covering parsing of the FEO_EXPORTS JSON, including valid, empty, invalid, and single-export cases.

### CODE REVIEW

File: rest/util/createChromeConfiguration.go [hunk 1]
```diff
@@ -562,6 +563,23 @@ func parseApiSpec(apiSpecVar string, env string) ([]byte, error) {
...
 }
 
+func parseExports(exportsVar string) ([]byte, error) {
+	var exports []interface{}
+
+	if exportsVar == "" {
+		logrus.Warn("FEO_EXPORTS is not set, using empty configuration")
+		exports = []interface{}{}
+	} else {
+		err := json.Unmarshal([]byte(exportsVar), &exports)
```
Comment (catastrophe-brandon) [inline rest/util/createChromeConfiguration.go:573]: For clarification, the unmarshal operation is just a validation check for the soundness of the json to confirm it can be de-serialized?

**Review by catastrophe-brandon — APPROVED** [review]

File: rest/util/createChromeConfiguration.go (hunk 1 above)
Comment (justinorringer) [reply rest/util/createChromeConfiguration.go:573]: Yup! exactly
